### PR TITLE
chore(docs): remove duplication of "nav-item"

### PR DESCRIPTION
**Pull request by TitanFighter**
`nav-item` in `extra (contentless) tab buttons should be a <b-nav-item> or have the class nav-item with a root element of <li> and class nav-item for proper rendering and semantic markup.` mentioned twice. Lets fix it.

### Describe the PR

A clear and concise description of what the pull request does.

### PR checklist

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Enhancement
- [ ] ARIA accessibility
- [x] Documentation update
- [ ] Other (please describe)

**Does this PR introduce a breaking change?** (check one)

- [x] No
- [ ] Yes (please describe)

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch, **not** the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (i.e. `[...] (fixes #xxx[,#xxx])`, where "xxx" is the issue number)
- [x] It should address only one issue or feature. If adding multiple features or fixing a bug and adding a new feature, break them into separate PRs if at all possible.
- [x] The title should follow the [**Conventional Commits**](https://www.conventionalcommits.org/) naming convention (i.e. `fix(alert): not alerting during SSR render`, `docs(badge): update pill examples, fix typos`, `chore: fix typo in README`, etc). **This is very important, as the `CHANGELOG` is generated from these messages.**

**If new features/enhancement/fixes are added or changed:**

- [x] Includes documentation updates (including updating the component's `package.json` for slot and event changes)
- [ ] New/updated tests are included and passing (if required)
- [ ] Existing test suites are passing
- [x] The changes have not impacted the functionality of other components or directives
- [ ] ARIA Accessibility has been taken into consideration (Does it affect screen reader users or keyboard only users? Clickable items should be in the tab index, etc.)

**If adding a new feature, or changing the functionality of an existing feature, the PR's
description above includes:**

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

### CODE REVIEW

File: src/components/tabs/README.md [hunk 1]
```diff
@@ -303,8 +303,7 @@ If you want to add extra tabs that do not have any content, you can put them in
 <!-- b-tabs-item-slot.vue -->
 ```
 
-**Note:** extra (contentless) tab buttons should be a `<b-nav-item>` or have the class `nav-item`
-with a root element of `<li>` and class `nav-item` for proper rendering and semantic markup.
+**Note:** extra (contentless) tab buttons should be a `<b-nav-item>` or have a root element of `<li>` and class `nav-item` for proper rendering and semantic markup.
```
Comment (tmorehouse) [inline src/components/tabs/README.md]: Line needs to be wrapped at 100 characters to make prettier happy

**Comment by tmorehouse** [comment]
Thanks!.  if you can make the one minor change (max line length is 100 characters, so it needs a line break)

File: src/components/tabs/README.md (hunk 1 above)
Comment (TitanFighter) [reply src/components/tabs/README.md]: Done

Comment (tmorehouse) [reply src/components/tabs/README.md]: I should have been a bit more clear... "as many words that will fit into a 100 character width (with no trailing spaces)"

:wink:

**Pull request merged**